.circleci/config.yml

```diff
@@ -240,10 +240,12 @@ workflows:
     jobs:
       - hmpps/npm_outdated:
           slack_channel: << pipeline.parameters.alerts-slack-channel >>
+          node_tag: << pipeline.parameters.node-version >>
```

Comment:
these were defaulting to node 16 rather than node 20 which is what we're currently on!